openc3/lib/openc3/models/model.rb

```diff
@@ -85,10 +85,8 @@ def self.set(json, scope:)
     def self.from_json(json, scope:)
       json = JSON.parse(json, :allow_nan => true, :create_additions => true) if String === json
       raise "json data is nil" if json.nil?
-
       json[:scope] = scope
-      json.transform_keys!(&:to_sym)
-      self.new(**json, scope: scope)
+      self.new(**json.transform_keys(&:to_sym), scope: scope)
```

Comment:
I changed this to prevent from_json from directly modifying the passed in json arg (which is a Ruby has). This caused a bug where suddenly the hash I was using had transformed the keys to symbols.